docs/swagger.yaml

```diff
@@ -3968,6 +3968,213 @@ paths:
           description: User have no permission to list webhook jobs of the project.
         '500':
           description: Unexpected internal errors.
+
+  '/retentions/metadatas':
+    get:
+      summary: Get Retention Metadatas
+      description: Get Retention Metadatas.
+      tags:
+        - Products
+      responses:
```

Comment:
No any error cases?

Reply:
This just return string, no error cases

Reply:
I mean, for API, we should handle the error cases, like 400, 404, 403, 409, 500 etc.